src/main/MQ2Utilities.cpp

```diff
@@ -5154,6 +5154,9 @@ int GetHighestAvailableBagSlot()
 
 	int highestInvSlot = InvSlot_LastBonusBagSlot;
 
+	if (pLocalPC->bGM)
+		return highestInvSlot;
```

Comment:
Can you add a comment that explains why we care about GM flag (its for emu?)

Reply:
Hope that was good enough